website/src/components/HourglassIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import type { WithOptional } from '@edgeandnode/common'
+import { ExperimentalIcon as Icon, type ExperimentalIconProps as IconProps } from '@edgeandnode/gds'
+
+interface HourglassIconProps extends WithOptional<IconProps, 'alt'> {
+  fillPercent?: number
```

Comment:
@0xa3k5 – Thanks for changing the name, but the functionality should also change to work as a percentage. :P I can do that.

Reply:
But why 15? That seems like a completely arbitrary number. 🤔

Reply:
it is completely arbitrary, i just made it up. we don't have many documents where the reading time is >15 so its a easy enough general grouping

Reply:
Ok so it sounds like this value should be the reading time in minutes? I don't think this component should know about that at all, especially if we move it to GDS. That logic (converting the reading time to icon fill %) should be in the consumer.

Reply:
Ah, well I think both of them could potentially be used elsewhere (like in the blog – I know I always give that example but docs and blog are similar in a few ways haha). And even if it wasn't moved, I feel like accepting a fill % would be better code, but if you disagree, at the very least the prop name should say what it is, so `readingTime` (with a comment that it's in minutes).

Reply:
i didnt mean that to be a very precise indication, its just something extra that vast majority of the people will not even notice its there. feel free to make any necessary changes as you deem fit on the pr

Reply:
Ok, sorry for being nitpicky, I just thought `value` was unclear and then 15 appeared kind of a magic number in the component. I will move these to GDS and update your PR, and maybe we should also use the new Button instead of Link for the "Edit on GitHub" link like you said?

Reply:
agree but i feel like they should not be blockers for this PR, this is ready to be used, could we merge it?

Reply:
Agreed, let's just change this name to `readingTime`, add `TODO`s for moving the icons to GDS, and merge?